src/libs/Permissions.js

```diff
@@ -51,19 +51,10 @@ function canUseDefaultRooms(betas) {
     return _.contains(betas, CONST.BETAS.DEFAULT_ROOMS) || canUseAllBetas(betas);
 }
 
-/**
- * @param {Array<String>} betas
- * @returns {Boolean}
- */
-function canUseInternationalization(betas) {
-    return _.contains(betas, CONST.BETAS.INTERNATIONALIZATION) || canUseAllBetas(betas);
```

Comment:
Can you also remove this beta from const.js if it is unused now?

Reply:
Updated